ide/gsf.testrunner.ui/apichanges.xml

```diff
@@ -64,6 +64,19 @@
         </description>
         <class package="org.netbeans.modules.gsf.testrunner.ui.api" name="TestMethodController"/>
     </change>
+    <change id="ComputeTestMethodsAndTestMethodGetTestClassPosition">
+        <api name="CommonTestrunnerUIAPI"/>
+        <summary>Added ComputeTestMethods SPI and getTestClassPosition() to TestMethod</summary>
+        <version major="1" minor="25"/>
+        <date day="26" month="6" year="2021"/>
+        <author login="dbalek"/>
+        <compatibility addition="yes"/>
+        <description>
+            Added SPI to provide a list of {@link TestMethod}s found in a parsed {@link Source}s.
```

Comment:
`{@link}` is unlikely to work. Use `<a href="@TOP@/pkg/TestMethod.html">`

Reply:
Fixed.